index.src.html

```diff
@@ -4133,6 +4136,40 @@
 : <dfn event for=ImageTrack>change</dfn>
 :: Fired at the {{ImageTrack}} when the {{ImageTrack/frameCount}} is altered.
 
+Resource Reclamation{#resource-reclamation}
+==============================================
+
+When resources are constrained, a User Agent might want to pro-actively reclaim
+codecs. This is particularly true in the case where hardware codecs are limited,
+and shared accross all web pages.
+
+A User Agent is allowed to [=reclaim a codec=], freeing the resources of a
+<dfn>reclaimable codec</dfn>, which is a defined as a codec that is either an
+[=inactive codec=] or a [=background codec=].
+
+An <dfn>inactive codec</dfn> is a codec that has not received a call to
+`encode()` or `decode()` in the past 1 second, and has no pending outputs.
+
+A <dfn>background codec</dfn> is a codec whose {{ownerDocument}}'s
+{{Document/hidden}} attribute is {{hidden|"hidden"}}.
+
+When selecting which [=reclaimable codec=] to evict, the User Agent should
+prioritize, in order, the following criteria:
+    1. What type of resource is the codec holding on to? User Agents must only
```

Comment:
Don't like 'must' here.

Reply:
Removed.

I am purposely leaving "resource" undefined, meaning that this could be HW codec slots, memory, CPU, power. Should these be called out?

Reply:
Here and elsewhere I would refer to the existing dfn of (codec) system resources
https://w3c.github.io/webcodecs/#system-resources